README.md

```diff
@@ -27,6 +27,38 @@ create forms by leveraging the capabilities of JSON and JSON schema.
     <jsonforms schema="vm.schema" uischema="vm.uiSchema" data="vm.data"></jsonforms>
   </div>
   ```
+6. Use Bootstrap version:
+
+  In order to use the bootstrap version you need to load the `bootstrap.css`, furthermore you need the `jsonforms-bootstrap.css` instead of the `jsonforms.css`:
+  ```html
+  <link rel="stylesheet" href="/node_modules/bootstrap/dist/css/bootstrap.css"/>
+  <link rel="stylesheet" href="/node_modules/jsonforms/dist/jsonforms-bootstrap.css"/>
+  ```
+  You also need to load the `jsonforms-bootstrap.js` after the `jsonforms.js`:
```

Comment:
Update to: You'll also need to load an additional JS file `jsonforms-bootstrap.js` in order to fully integrate JSONForms with Bootstrap.